src/cgame/cg_main.cpp

```diff
@@ -1009,6 +1019,14 @@ cvarTable_t cvarTable[] =
 	{ &etj_jumpSpeedsShowDiff, "etj_jumpSpeedsShowDiff", "0", CVAR_ARCHIVE },
 	{ &etj_jumpSpeedsFasterColor, "etj_jumpSpeedsFasterColor", "0.0 1.0 0.0 1.0", CVAR_ARCHIVE },
 	{ &etj_jumpSpeedsSlowerColor, "etj_jumpSpeedsSlowerColor", "1.0 0.0 0.0 1.0", CVAR_ARCHIVE },
+	// Strafe quality
+	{ &etj_drawStrafeQuality, "etj_drawStrafeQuality", "1", CVAR_ARCHIVE },
+	{ &etj_strafeQualityX, "etj_strafeQualityX", "100", CVAR_ARCHIVE },
+	{ &etj_strafeQualityY, "etj_strafeQualityY", "100", CVAR_ARCHIVE },
```

Comment:
I'd rather have these as relative than absolute values, I know it's nowhere consistent in the codebase but relative values make sure they are by default on same place regardless of screen aspect ratio.